pypsa/optimization/constraints.py

```diff
@@ -62,19 +62,64 @@ def define_operational_constraints_for_non_extendables(
         dispatch_lower = (1, dispatch_lower), (-1, loss)
         dispatch_upper = (1, dispatch_upper), (1, loss)
     n.model.add_constraints(
-        dispatch_lower, ">=", lower, name=f"{c}-fix-{attr}-lower", mask=active
+        dispatch_lower, ">=", lower, name=f"{c}-fix-non-comm-{attr}-lower", mask=active
```

Comment:
Better not to change existing function names and constraints name for consistency with the past; I'd recommend to revert these changes, including the name of the function

Reply:
[PR_comment.docx.pdf](https://github.com/user-attachments/files/16735958/PR_comment.docx.pdf)